src/components/KMultiselect/KMultiselect.vue

```diff
@@ -713,7 +617,7 @@ const handleMultipleItemsSelect = (items: MultiselectItem[]) => {
   stageSelections()
 }
 
-const handleMultipleItemsDeselect = (items: MultiselectItem[], restage = false) => {
+const handleMultipleItemsDeselect = (items: MultiselectItem<Value>[], restage = false) => {
```

Comment:
```suggestion
const handleMultipleItemsDeselect = (items: Items, restage = false) => {
```

Or shall we define `type Item = MultiselectItem<Value>` and use `Item[]` where needed?